db/seeds.rb

```diff
@@ -24,7 +24,32 @@
 	description: "my description of Futurama",
 	small_cover_url: "/tmp/futurama.jpg",
 	large_cover_url: "/tmp/futurama.jpg",
-	category_id: 3)
+	category_id: 2)
```

Comment:
You are still signed in to your app, meaning the session still has a user id of 1 set, but you cleared your users. The fix is to clear your cookies for that domain or for all domains